www/CQMContext.js

```diff
@@ -85,6 +121,55 @@ Entity.prototype._getImage = function(width, height, endpoint, successCallback,
 };
 
 /**
+ * This represents the an entity list.
+ * @constructor
+ */
+var EntityList = function(entityList) {
+	this.entities = this._processRawEntities(entityList.entities);
+	this.hasNextPage = entityList.hasNextPage;
+	this._parentId = entityList.parentId;
+	this._parentVersion = entityList.parentVersion;
+	this._parentVisibilityHash = entityList.parentVisibilityHash;
+};
+
+/**
+ * Get next server page of entities for this EntityList
+ *
+ * @param {Function} successCallback The function to call when the next server page of entities is available.
+ * @param {Function} errorCallback The function to call when there is an error getting the next server page of entities. (OPTIONAL)
+ */
+EntityList.prototype.getNextPage = function(successCallback, errorCallback) {
+	argscheck.checkArgs('fF', 'EntityList.getNextPage', arguments);
+	if (this.hasNextPage == false) {
+		errorCallback && errorCallback(new CQMContextError(CQMContextError.NO_MORE_ENTITY_PAGES_ERROR));
+		return;
+	}
+	var success = successCallback && function(entityList) {
+			this.entities = this._processRawEntities(entityList.entities);
+			this.hasNextPage = entityList.hasNextPage;
+			successCallback(entityList);
+		};
+
+	var fail = errorCallback && function(code) {
+    	var ce = new CQMContextError(code);
+        errorCallback(ce);
+    };
```

Comment:
I think 4-spaces is the practice in most of the cordova plugins. For example, see https://github.com/apache/cordova-plugin-device/blob/master/www/device.js